src/js/utils/string-utils.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Useful functions for dealing with strings.
+ */
+(function(define) {
+    'use strict';
+    define(['underscore', './string-utils.js'], function(_, StringUtils) {
+        var interpolate;
+
+        /**
+         * Returns a string created by interpolating the provided parameters.
+         *
+         * The HTML text is provided as a tokenized format string where parameters
```

Comment:
@andy-armstrong it looks like you forgot to update the documentation after this method moved out of HTMLUtils? This isn't HTML text, it's just a string, correct?